zooniverse/front-end-monorepo
# Made SubjectViewer props viewBox and src optional

Package: `lib-classifier`

Closes #1999 

Describe your changes:
`viewBox` in SingleImageViewer and `src` in SVGPanZoom are now optional.
A lot of Prettier format changes.

To test:
1. In the front-end-monorepo dir, run yarn bootstrap
2. cd to packages/lib-classifier and run yarn dev
3. In a browser, go to https://localhost:8080/
4. Check browser dev tools/console. No more errors! (See Issue #1999 for error details)


# Review Checklist

## General

- [ ] Is the documentation up to date?

## Components
- [ ] Has a storybook story been created or updated?
- [ ] Is the component accessible? 
  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - [ ] Is it passing accessibility checks in the storybook?

## Apps

- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?

## Publishing

- [ ] Is the changelog updated?
- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?

## Post-merging

- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
- [ ] Is the new feature working or bug now fixed?
  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
  - [ ] Is this approved by our designer?
- [ ] Is this ready for production deployment?
